mgmt/api/CoreAPIShared.h

```diff
@@ -84,4 +84,9 @@ int connectDirect(const char *host, int port, uint64_t timeout);
 int get_event_id(const char *event_name);
 char *get_event_name(int id);
 
+enum class HostStatusCmd_t {
```

Comment:
Is this necessary? It seems like your using this to split `UP` and `DOWN` within `HostStatusSet` but it's tying in `CoreAPIShared.h` to a bunch of classes that don't really need it. 

You have two separate functions `TSHostStatusSetUp` and `TSHostStatusSetDown` I think it would be better to do the same. So something like `HostStatusSetUp` and `HostStatusSetDown`. 

This way you don't need `HostStatusCmd_t` and can stick with just using `OpType::HOST_STATUS_DOWN` and `OpType::HOST_STATUS_UP` instead of carrying this `HostStatusCmd_t` all the way through and then matching it back to a `OpType`

Reply:
@xf6wang No it's not necessary and thanks for pointing this out.  I'll fix this.